docs/monitoring/docs/021-gs-monitoring-custom-metrics.md

```diff
@@ -0,0 +1,149 @@
+
+This example has taken a basic example where `Gauge` and `Counter` metrics are exported using the package `prometheus`.
+
+1. Deploy the sample metrics application
+    ```bash
+    kubectl apply -f https://raw.githubusercontent.com/kyma-project/examples/master/monitoring-custom-metrics/deployment/deployment.yaml
+    
+    kubectl apply -f https://raw.githubusercontent.com/kyma-project/examples/master/monitoring-custom-metrics/deployment/service-8080.yaml
+    
+    kubectl apply -f https://raw.githubusercontent.com/kyma-project/examples/master/monitoring-custom-metrics/deployment/service-8081.yaml
+
+    kubectl apply -f https://raw.githubusercontent.com/kyma-project/examples/master/monitoring-custom-metrics/deployment/service-monitor.yaml
+    ```
+
+    ```bash
+    kubectl get pods
+    NAME                             READY     STATUS    RESTARTS   AGE
+    sample-metrics-c9f998959-jd2fz   2/2       Running   0          2m
+    sample-metrics-c9f998959-kfbp8   2/2       Running   0          2m
+    sample-metrics-c9f998959-nnp2n   2/2       Running   0          2m
+    sample-metrics-c9f998959-vdnkn   2/2       Running   0          2m
+    ```
+
+2. Run the `port-forward` on the sample-metrics-8081 Service for port 8081 to check the metrics
+    ```bash
+    kubectl port-forward svc/sample-metrics-8081 8081:8081
+    ```
+    Then open a browser and access [`http://localhost:8081/metrics`](http://localhost:8081/metrics)
+    ![metrics on port 8081](assets/sample-metrics-2.png)
+
+The source code for the sample application is [here](https://github.com/kyma-project/examples/blob/master/monitoring-custom-metrics/main.go).
+
+- See [package prometheus](https://godoc.org/github.com/prometheus/client_golang/prometheus) for its reference documentation.
+
+- Learn more on [prometheus metric types](https://prometheus.io/docs/concepts/metric_types/).
+
+### Access the exposed metrics in Prometheus
+
+ Run the `port-forward` command on the `core-prometheus` service:
+    
+```bash
+kubectl port-forward svc/core-prometheus -n kyma-system 9090:9090
+Forwarding from 127.0.0.1:9090 -> 9090
+Forwarding from [::1]:9090 -> 9090
+```
+All the **sample-metrics** endpoints appear as the [`Targets`](http://localhost:9090/targets#job-sample-metrics-8081) list.
+![Prometheus Dashboard](assets/pm-dashboard-1.png)
+
+Use either `cpu_temperature_celsius` or `hd_errors_total` in the **expression** field in [here](http://localhost:9090/graph) and click the `Execute` button to check the values scrapped by Prometheus.
+![Prometheus Dashboard](assets/pm-dashboard-2.png)
+
+### ServiceMonitor
+
+With the help of ServiceMonitor, Prometheus is able to reach the service. A ServiceMonitor is a specific CRD used by the prometheus operator to monitor services.
+
+In Kyma the prometheus server discovers all ServiceMonitors through a ```serviceMonitorSelector:``` matching the label ```prometheus: core```
```

Comment:
``In Kyma, the Prometheus server discovers all ServiceMonitors through the **serviceMonitorSelector** matching the `prometheus: core` label.``

Reply:
done